components/landing/CuratedList.vue

```diff
@@ -70,17 +80,20 @@ const curatedCollection = [
 @Component<CuratedList>({
   components,
 })
-export default class CuratedList extends mixins(AuthMixin) {
-  private collections: [] = []
+export default class CuratedList extends mixins(AuthMixin, PrefixMixin) {
+  protected collections: Collection[] = []
 
   async fetch() {
+    const query = await resolveQueryPath(
+      this.urlPrefix,
+      'collectionCuratedList'
+    )
     const result = await this.$apollo
       .query<any>({
-        query: collectionCuratedList,
-        client: 'subsquid',
-        variables: {
-          list: curatedCollection,
-        },
+        query: query.default,
+        client: this.urlPrefix === 'rmrk' ? 'subsquid' : this.urlPrefix,
+        variables:
+          this.urlPrefix === 'rmrk' ? { list: curatedCollection } : Unknown,
```

Comment:
what's unknown type?

Reply:
lol I dunno, needed a way to remove gql `variables` for !rmrk 
maybe it work with `undefined`
Later this will be changed with bsx collection List Id